### PR TITLE
Minimal entity typings (aliases and identifiers)

### PR DESCRIPTION
**Pull request by kellnerd**
Incomplete type definitions for entities with aliases and identifiers which were written for https://github.com/metabrainz/bookbrainz-site/pull/943 and could be used in other places too.

As long as the data models themselves are untyped, it helps to have at least some basic, separate type definitions for certain parts which are defined in a central place. They can be expanded as needed until the models have been migrated to a type-safe ORM ([BB-729](https://tickets.metabrainz.org/browse/BB-729)).
It should be trivial to integrate these provisional type definitions into the new models then.

### CODE REVIEW

File: src/types/entity.ts
Comment (kellnerd) [inline src/types/entity.ts:30]: This type is already defined elsewhere, but I think this would be the better place for this definition.
If you agree, I will delete the older definition and point all import statements to the new one.
https://github.com/metabrainz/bookbrainz-data-js/blob/8af33f049a292e6632938cd6c53e1b2a81d9af43/src/func/types.ts#L25-L26

File: src/types/aliases.ts
Comment (kellnerd) [inline src/types/aliases.ts:27]:
```suggestion
	languageId: number | null,
```
I haven't checked yet whether it makes more sense to mark these columns as nullable instead of optional.

**Review by MonkeyDo — APPROVED** [review]
Good for a first go !

I'll let you do the `EntityTypeString` move before merging.

File: src/types/aliases.ts
Comment (kellnerd) [reply src/types/aliases.ts:27]: Ok, I had a look again at the JSON samples which I had extracted from the React initial page properties a while ago. I did not find an alias without language, but all occurrences of the columns which I marked as optional in this PR were always present and filled with `null` in my samples if they have no other value.

No idea why I initially decided to mark them as optional, but I will change these to be nullable.

**Comment by kellnerd** [comment]
I've now changed the module specifiers for all imports of `EntityTypeString` (also in bb-site, but I still have to push this commit, together with the other changes to that PR).

The column types have been changed to be nullable. Marking them as optional only makes sense if we want to be able to insert incomplete records into the DB, but we are currently using the
types only for fetched records (which should always be complete).
If we should decide to use these types to insert/update rows later, we can still use derived types, e.g. by using `Partial<T>`.

**Comment by MonkeyDo** [comment]
> If we should decide to use these types to insert/update rows later, we can still use derived types, e.g. by using `Partial<AliasT>`.

Great, thanks for updating the PR!
For updates, I think the current types will work fine, if I read the docs correctly yesterday we would need to explicitly set the value to `null` to set a `NULL` value in the column.
For inserts as you said we can use `Partial` types.

**Pull request merged**
